### ISSUE TITLE
accessibility  audio cannot make out numbers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [queueit/recaptcha#251]
```
What steps will reproduce the problem?
1. reduce background noise
2. tell user how many numbers they are looker for.
3.

What is the expected output? What do you see instead?
Users dependent on audio can make outnumbers

What version of the product are you using? On what operating system?
N/A

Please provide any additional information below.


```

Original issue reported on code.google.com by `njohnso...@gmail.com` on 1 Jul 2015 at 4:55
